src/Symfony/Component/Intl/Transliterator/EmojiTransliterator.php

```diff
@@ -11,144 +11,149 @@
 
 namespace Symfony\Component\Intl\Transliterator;
 
-if (\PHP_VERSION_ID >= 80200) {
-    final class EmojiTransliterator extends \Transliterator
-    {
-        use EmojiTransliteratorTrait;
-
-        private const QUICK_CHECK = "\xA9\xAE\xE2\xE3\xF0";
-        private const REVERSEABLE_IDS = [
-            'emoji-github' => 'github-emoji',
-            'emoji-slack' => 'slack-emoji',
-            'github-emoji' => 'emoji-github',
-            'slack-emoji' => 'emoji-slack',
-        ];
-
-        public readonly string $id;
-    }
+if (!class_exists(\Transliterator::class)) {
+    throw new \LogicException(sprintf('You cannot use the "%s\EmojiTransliterator" class as the "intl" extension is not installed. See https://www.php.net/intl', __NAMESPACE__));
```

Comment:
no need for www

Reply:
You want me to delete internet? 🤔 
Oh got it 😂